packages/core/src/sync-store/postgres/store.ts

```diff
@@ -1303,7 +1309,7 @@ export class PostgresSyncStore implements SyncStore {
 
     exprs.push(
       eb(
-        "logs.address",
+        sql`lower(logs.address)`,
```

Comment:
If we don't also lowercase the generated `childAddress` before we compare, I think we're counting on the log `topic1,2,3` and `data` columns to be stored as lowercase.

Reply:
Yes, I think there are many spots where we are counting on return values to be lowercase. Let's talk about this more because it might make sense to make this a separate pr.